tasks/xpack/security/elasticsearch-xpack-activation.yml

```diff
@@ -14,6 +15,24 @@
     license_activated.status != 200 or
     license_activated.json.license_status is not defined or
     license_activated.json.license_status != 'valid'
+  when: es_version is version('6.9', operator='<=')
+
+- name: Activate ES license (with security authentication)
+  uri:
+    method: PUT
+    url: "{{es_api_scheme}}://{{es_api_host}}:{{es_api_port}}/_license?acknowledge=true"
+    user: "{{es_api_basic_auth_username  | default(omit)}}"
+    password: "{{es_api_basic_auth_password | default(omit)}}"
+    body_format: json
+    body: "{{ es_xpack_license }}"
+    return_content: yes
```

Comment:
You are going to need to either point to the actual certs for this request or disable the validation with: `validate_certs: no`.